diff --git a/cmd/kubeadm/app/constants/constants.go b/cmd/kubeadm/app/constants/constants.go
index afd5ce5..7b80d72 100644
--- a/cmd/kubeadm/app/constants/constants.go
+++ b/cmd/kubeadm/app/constants/constants.go
@@ -296,7 +296,7 @@ var (
 	MinimumKubeletVersion = version.MustParseSemantic("v1.10.0")
 
 	// MinimumKubeletConfigVersion specifies the minimum version of Kubernetes where kubeadm supports specifying --config to the kubelet
-	MinimumKubeletConfigVersion = version.MustParseSemantic("v1.11.0-alpha.1")
+	MinimumKubeletConfigVersion = version.MustParseSemantic("v1.13.0-alpha.1")
 
 	// SupportedEtcdVersion lists officially supported etcd versions with corresponding kubernetes releases
 	SupportedEtcdVersion = map[uint8]string{
